dbtmetabase/_exposures.py

```diff
@@ -346,6 +354,11 @@ def __format_exposure(
             + f"Created On: __{created_at}__"
         )
 
+        if average_query_time_ms:
+            average_query_time_s = average_query_time_ms // 1000
+            average_query_time = f"{int(average_query_time_s // 60)}:{str(int(average_query_time_s % 60)).zfill(2)}"
```

Comment:
If the original value is in milliseconds (i.e. 1000th), why are you zfill'ing with only 2 and not 3?